arangod/Aql/EnumerateCollectionExecutor.h

```diff
@@ -63,26 +63,25 @@ class EnumerateCollectionExecutorInfos : public ExecutorInfos {
   ~EnumerateCollectionExecutorInfos() = default;
 
   ExecutionEngine* getEngine() { return _engine; };
-  Collection const* getCollection() { return _collection; };
-  Variable const* getOutVariable() { return _outVariable; };
-  std::vector<std::string> const& getProjections() { return _projections; };
-  transaction::Methods* getTrxPtr() { return _trxPtr; };
-  std::vector<size_t> const& getCoveringIndexAttributePositions() {
+  Collection const* getCollection() const { return _collection; }
+  Variable const* getOutVariable() const { return _outVariable; }
+  std::vector<std::string> const& getProjections() const noexcept { return _projections; }
+  transaction::Methods* getTrxPtr() const { return _trxPtr; }
+  std::vector<size_t> const& getCoveringIndexAttributePositions() const noexcept {
     return _coveringIndexAttributePositions;
-  };
-  bool getProduceResult() { return _produceResult; };
-  bool getUseRawDocumentPointers() { return _useRawDocumentPointers; };
-  bool getRandom() { return _random; };
-  RegisterId getOutputRegisterId() { return _outputRegisterId; };
+  }
+  bool getProduceResult() const { return _produceResult; }
+  bool getUseRawDocumentPointers() const { return _useRawDocumentPointers; }
+  bool getRandom() const { return _random; }
+  RegisterId getOutputRegisterId() const { return _outputRegisterId; }
 
  private:
   RegisterId _outputRegisterId;
```

Comment:
`RegisterId` is an `unsigned int` (i.e., usually 32bit), so it should be moved before the bools.